### ISSUE TITLE
refactor epw loading

### ISSUE THREAD

**Issue by devngc** [pollination/sample-apps#12]
At more than one place, we're loading epw files. We should start thinking of modularizing the code to avoid repetition. Should be assigned to me.

**Comment by mostaphaRoudsari** [comment]
I agree and there will be more of these. I think we should add these functionalities in its own library called `ladybug-streamlit` or something similar that exposes these frequent functionalities in easy-to-use functions.
